provisioning_templates/snippet/http_proxy.erb

```diff
@@ -0,0 +1,12 @@
+<%#
```

Comment:
I think so too, but I wanted to sync it with this repo, we can remove it separately if that makes sense (some people might use it though)